<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>596</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>596</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DirAge2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionRescan"/>
   <addaction name="actionCancel"/>
   <addaction name="actionSaveReport"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="folder-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRescan">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Rescan</string>
   </property>
   <property name="toolTip">
    <string>Rescan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionOpenFromView">
   <property name="icon">
    <iconset theme="folder-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Shift+Return</string>
   </property>
  </action>
  <action name="actionOpenFromViewInFM">
   <property name="text">
    <string>Open in File Manager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionScaleLinear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linear Scale</string>
   </property>
  </action>
  <action name="actionScaleSqrt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale by sqrt</string>
   </property>
  </action>
  <action name="actionScaleLn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale by ln</string>
   </property>
   <property name="toolTip">
    <string>Scale by log10</string>
   </property>
  </action>
  <action name="actionSearchFixed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ABC</string>
   </property>
   <property name="toolTip">
    <string>Search by String</string>
   </property>
  </action>
  <action name="actionSearchWildcard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>*?</string>
   </property>
   <property name="toolTip">
    <string>Search by Wildcard Pattern</string>
   </property>
  </action>
  <action name="actionSearchRegex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>^.</string>
   </property>
   <property name="toolTip">
    <string>Search by Regular Expression</string>
   </property>
  </action>
  <action name="actionSearchNext">
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSearchPrevious">
   <property name="text">
    <string>Find Previous</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="text">
    <string>Expand All</string>
   </property>
  </action>
  <action name="actionExpandCollapseSiblingsToLevel">
   <property name="text">
    <string>Collapse Siblings to Level</string>
   </property>
  </action>
  <action name="actionSaveReport">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Report as JSON</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
